src/Microsoft.TestPlatform.Extensions.TrxLogger/TrxLogger.cs

```diff
@@ -33,30 +33,30 @@ namespace Microsoft.VisualStudio.TestPlatform.Extensions.TrxLogger
     [ExtensionUri(TrxLoggerConstants.ExtensionUri)]
     public class TrxLogger : ITestLoggerWithParameters
     {
-        #region Fields
-
         #region Constructor
 
         /// <summary>
         /// Initializes a new instance of the <see cref="TrxLogger"/> class.
         /// </summary>
-        public TrxLogger():
-            this (new Utilities.Helpers.FileHelper())
-        {
-        }
+        public TrxLogger() : this(new Utilities.Helpers.FileHelper()) { }
 
         /// <summary>
         /// Initializes a new instance of the <see cref="TrxLogger"/> class.
         /// Constructor with Dependency injection. Used for unit testing.
         /// </summary>
         /// <param name="fileHelper">The file helper interface.</param>
-        protected TrxLogger(IFileHelper fileHelper)
+        protected TrxLogger(IFileHelper fileHelper) : this(new Utilities.Helpers.FileHelper(), new InternalFileHelper()) { }
+
+        internal TrxLogger(IFileHelper fileHelper, InternalFileHelper internalFileHelper)
```

Comment:
Why can't those be the same class?

Reply:
The other class is public, and in a different assembly.

Reply:
Hmm right. Can we call it TrxFileHelper instead of internal please? It will make it easier to know what project it belongs to.